apps/condo/domains/news/utils/clientSchema/NewsRenders.tsx

```diff
@@ -80,19 +82,20 @@ const POSTFIX_PROPS: TextProps = { type: 'secondary', style: { whiteSpace: 'pre-
 
 export const ResendNewsButton = ({ intl, newsItem }) => {
     const ResendMessage = intl.formatMessage({ id: 'pages.condo.news.resend' })
-
-    //TODO(DOMA-5917) this functionality will be implemented as part of next task
+    
     const handleClick = useCallback((e) => {
         e.stopPropagation()
     }, [])
     
 
     return (
-        <Tooltip title={ResendMessage} placement='bottomLeft'>
-            <div style={RESEND_MESSAGE_CONTAINER_STYLE} onClick={handleClick}>
-                <RefreshCw size='small'/>
-            </div>
-        </Tooltip>
+        <Link key='update' href={`/news/${get(newsItem, 'id')}/update`}>
```

Comment:
Just FYI.
Resend means creating a new NewsItem with the same parameters. We just need to open a prefilled form.

Reply:
yes, that's what's happening now, it's just that even if I create new NewsItem with the same parameters, I do it through UpdateNewsPageContent component, as far as I understand it would be better to transfer this CreateNewsPageContent logic for better code readability, right?

Reply:
ok, I just want to be sure that frontend logic will work properly. Anyway, "update" it's about existing items.